app/views/wrangling_guidelines/show.html.erb

```diff
@@ -1,5 +1,5 @@
 <!--Descriptive page name, messages and instructions-->
-<h2 class="heading"><%= link_to ts('Wrangling Guidelines'), wrangling_guidelines_path %> > <%= @wrangling_guideline.title %></h2>
```

Comment:
Is this `>` the typo you're referring to? It's not a typo -- it's part of the heading, to make it breadcrumb-like. For example, "[Wrangling Guidelines](https://archiveofourown.org/wrangling_guidelines) > Wrangling Guidelines - Intro and General Concepts" on [these guidelines](https://archiveofourown.org/wrangling_guidelines/2).

Reply:
whoops, I'll fix that...